### ISSUE TITLE
Create new playground app

### ISSUE THREAD

**Issue by ayoayco** [astro-reactive/astro-reactive#263]
In addition to our landing page and docs sites, we can now create a playground using https://webcontainers.io

**Comment by Preet-Sojitra** [comment]
Correct me if I am wrong. This webcontainers are used to create interactive code editors on web where user can play around and get to learn more?

**Comment by ayoayco** [comment]
Yeah exactly. I think we can find a lot of examples in other more mature libraries

**Comment by Preet-Sojitra** [comment]
On website they have put example of sveltekit. Sveltkit has implemented playground on their website. You can check it out.

**Comment by fazzaamiarso** [comment]
This is exciting! 

@ayoayco So with this, will we migrate our `demo` to playground? Like [Qwik](https://qwik.builder.io/examples/introduction/hello-world/) and [Svelte](https://svelte.dev/tutorial/basics)?

I'm interested to try out webcontainer.

**Comment by ayoayco** [comment]
Yes exactly @fazzaamiarso ... let me know what you find out. I'm not sure if it needs a StackBlitz account

**Comment by fazzaamiarso** [comment]
@ayoayco I think it's quite difficult to do it with Astro because webcontainer mostly works on the client side. We can use framework like React (most familiar with me) or any other like svelte, solid.

But I found [this playground](https://github.com/Jutanium/tutour) with Solid, which is pretty solid 🤣. I always wanted to try Solid though. If I'm not mistaken, the `hyperdrive` package will use solid too, right?

what do you think?

**Comment by fazzaamiarso** [comment]
I've build a minimal playground with vanilla-ts + vite. Tried using our library and it worked. Although, we definitely need a framework for our DX.

**Comment by ayoayco** [comment]
@fazzaamiarso I'm more excited to try svelte than react to be honest. :) I'm fine if you want to PR what you have now so we can iterate on improving it. But if you're also interested to add a framework I vote for svelte because I'm curious about it as well. Might be a nice new project.
